### PR TITLE
feat: ERC20/721/1155 enrich ephemeral.

### PR DESCRIPTION
**Pull request by TalkWIthKeyboard**

### CODE REVIEW

**Comment by onesuper** [comment]
Ephemeral models can be only used like CTE across dbt project. AFAIK, when a model is set as ephemeral, it will not appear in the database. The end users can not access it by name.

**Comment by onesuper** [comment]
And we could transform the right and accurate data in dbt_ethereum_source. If so, dbt_ethereum_source will be more helpful to others.

File: dbt_project.yml [hunk 1]
```diff
@@ -24,6 +26,10 @@ models:
         - level_modeling
         - proj_labels
 
+    enrich:
+      +schema: enrich
```
Comment (onesuper) [inline dbt_project.yml:30]: I think enrich is not a straight forward name. We could use common/enrich in staging tables.

 Could we name those derived models' schema like 'erc20/erc721/erc1155' like what Dune did?

**Comment by TalkWIthKeyboard** [comment]
> Ephemeral models can be only used like CTE across dbt project. AFAIK, when a model is set as ephemeral, it will not appear in the database. The end users can not access it by name.

Sure, I thought these tables would not be used directly by users. Creating views in the `dbt_ethereum_source` is better.

**Pull request closed without merging**